client/src/components/Ticketing/ticketingmanager/Doorlistpage/doorlist.tsx

```diff
@@ -156,7 +156,7 @@ const DoorList = () => {
           )}
         </select>
         <div className='text-4xl font-bold '>{`Showing: ${titleCase(eventName)}`}</div>
-        <div className='text-2xl font-bold text-zinc-700'>{`${date}, ${time}`}</div>
+        <div className='text-2xl font-bold text-zinc-700'>{date && time ? `${date}, ${time}` : `${date}${time}`}</div>
```

Comment:
Just a curious question but what was the reasoning behind this change?

Reply:
Oh wait, I got it, it's because when there isn't a date or a time, then only the comma would render.